index.d.ts

```diff
@@ -2,6 +2,26 @@
 import File = require('vinyl');
 import * as globStream from 'glob-stream';
 import * as through from 'through2';
+import {EventEmitter} from 'events';
+
+export interface WatchOptions {
+    interval?: number;
+    debounceDelay?: number;
+    cwd?: string;
+    maxListeners?: Function;
```

Comment:
I can't find it either. It also doesn't make much sense to me that it's supposed to be a function. I copied it over from the @types/vinyl-fs package but I can't find any reference to it in the JavaScript files. I guess I should remove it then.